herbert_core/classes/MPIFramework/@ClusterParpoolWrapper/ClusterParpoolWrapper.m

```diff
@@ -114,6 +114,18 @@
             
             cl  = parcluster();
             
+            % By default Matlab only utilises physical cores; enable use of
+            % logical cores if required
+            if cl.NumWorkers < obj.n_workers
```

Comment:
It's not obvious what this block of code is doing. `obj_n.workers` is the requested number, `cl.NumWorkers` is the number that will be created (default `num_physical_cores` if I understand correctly)